apps/api/src/server/applications/application/representations/publishable/publishable.route.js

```diff
@@ -0,0 +1,43 @@
+import { validateApplicationId } from '../../application.validators.js';
+import { asyncHandler } from '#middleware/async-handler.js';
+import { Router as createRouter } from 'express';
+import { getPublishableRepresentations } from './publishable.controller.js';
+
+const router = createRouter({ mergeParams: true });
+
+router.get(
+	'/',
+	/*
+            #swagger.tags = ['Applications']
+            #swagger.path = '/applications/{id}/representations/publishable'
+            #swagger.description = 'Gets publishable representations for an application'
+            #swagger.parameters['id'] = {
+                                    in: 'path',
+                                    description: 'Application ID',
+                                    required: true,
+                                    type: 'integer'
+            }
+            #swagger.responses[200] = {
+                    description: 'Representations',
+                    schema: {
+                            itemCount: 100,
+                            items:  [
+                                    {
+                                            id: 1,
```

Comment:
We seem to have a "representation" defined in a few places, it'd be better to do a definition once and reference it; or at least `/applications/{id}/representations` seems to return the same type.

Reply:
I've actually got the properties wrong in the swagger definition here. This new endpoint returns a smaller subset of the properties which `/applications/{id}/representations` returns.

I was in the processing of adding a single definition as you suggested, but given there's no common definition of representation (all the endpoints return different properties, some with related [contacts, etc] data) this might not make sense.